client/my-sites/exporter/export-card/advanced-settings.jsx

```diff
@@ -8,7 +8,7 @@ import { connect } from 'react-redux';
  * Internal dependencies
  */
 import PostTypeOptions from './post-type-options';
-import SpinnerButton from './spinner-button';
+import SpinnerButton from 'components/button/spinner-button';
```

Comment:
Moved this into `components/spinner-button` in 6f617fe, and fixed up the className